include/qpdf/QPDFObjGen.hh

```diff
@@ -39,7 +38,7 @@ class QPDFObjGen
     {
     }
     QPDF_DLL
-    QPDFObjGen(int obj, int gen) :
+    explicit QPDFObjGen(int obj, int gen) :
```

Comment:
Technically `explicit` is not useful here because the constructor takes two required args....